addons/crm/views/crm_lead_views.xml

```diff
@@ -736,6 +737,20 @@
             </field>
         </record>
 
+        <record id="crm_lead_view_pivot" model="ir.ui.view">
```

Comment:
I'm a bit concerned about all those changes.
The spec just specifies "add a bunch of field in 'invisible'.

Here, it looks like we move an entire view from "crm_enterprise" to here.
Was that discussed directly with the PO?